content/applications/general/email_communication/faq.rst

```diff
@@ -1,174 +1,208 @@
+  <email_communication/spf_compliant>` and/or :ref:`DKIM <email_communication/DKIM_compliant>`
+  configuration. Also, check to make sure the `mail.bounce.alias` is defined in the *System
+  Parameters*. Access *System Parameters* in :ref:`Developer mode <developer-mode>` by navigating to
+  :menuselection:`Settings App --> Technical Menu --> Parameters --> System Parameters`.
 
-Why is my email sent late?
-**************************
+Email sent late
+***************
 
-It may happen that you schedule an email campaign but it is not sent on time. We know that
-we use a delayed job to send emails that we consider as not urgent (Newsletters concept
-such as mass mailing, marketing automation, events). The system utility **cron** can be used
-to schedule programs to run automatically at predetermined intervals. We use that policy in order
-to avoid cluttering the mail servers and prioritize the communication.
+It may happen that an email campaign is scheduled, but isn't sent on time. It's known that  Odoo
+uses a delayed job to send emails that are considered as "not urgent" (Newsletters concept, such as:
+mass mailing, marketing automation, events). The system utility **cron** can be used to schedule
+programs to run automatically at predetermined intervals. Odoo uses that policy in order to avoid
+cluttering the mail servers and, instead, prioritizes the communication. This **cron** is called
+`Mail: Email Queue Manager`, and can be accessed in :ref:`Developer mode <developer-mode>` by going
+to :menuselection:`Settings App --> Technical Menu --> Automation --> Scheduled Actions`.
 
-The emails considered urgent (communication from one person to another one such as
-Sales Orders, Invoices, Purchase Orders, etc.) are sent directly.
+By default, the *Mass Mailing cron* runs every 60 minutes. This can be changed to no less than 5
+minutes. But, for the reasons mentioned above, this isn't recommended. Select the action
+name, `Mail: Email Queue Manager,` and proceed to make any necessary adjustments.
 
 .. image:: faq/email-scheduled-later.png
    :align: center
    :alt: Email scheduled to be sent later.
 
-By default, the Mass Mailing cron runs every 60 minutes. So, you should wait maximum an hour
-before the campaign is actually sent.
+Emails that are considered urgent (communication from one person to another, such as: Sales Orders,
+Invoices, Purchase Orders, etc.) are sent directly (immediately).
 
 Incoming emails
 ===============
 
-When you have an issue with incoming emails, there might not be an indication per se in Odoo.
-This is the client who tries to contact a database who will get a bounce (most of the
-time 550: mailbox unavailable).
+When there is an issue with incoming emails, there might not be an indication, per se, in Odoo. It
+is the sending email client, who tries to contact a database, that will get a bounce-back (most of
+the time a `550: mailbox unavailable`).
```

Comment:
This would be a message displayed on the screen, not a user input, so a :guilabel: is appropriate
```suggestion
the time a :guilabel:`550: mailbox unavailable`).
```